.cbmc-batch/jobs/aws_hash_iter_done/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+DEFINES += -DMAX_TABLE_SIZE=SIZE_MAX 
```

Comment:
Why do we need to bound to `SIZE_MAX`? I'd expect that removing this line would have no affect in the proof.

Reply:
Added comment to the makefile explaining why.